common/src/main/scala/explore/components/graphql/SubscriptionRender.scala

```diff
@@ -91,7 +90,7 @@ object SubscriptionRender {
       .componentWillUnmount { $ =>
         implicit val ce = $.props.ce
 
-        $.state.subscription.map(_.stop.runInCB).getOrEmpty
+        $.state.subscription.map(_.stop().runInCB).getOrEmpty
```

Comment:
Calling `stop` does not perform any side effects. Maybe I should remove the parenthesis in `clue`?

Reply:
I guess that could be a good alternative